Compile error in chem_driver (single precision)

It looks like TChem is hard-wiring a `real_type` typedef somewhere to `double`. Using GCC 9.3, I encounter the following error when trying to build `chem_driver.cpp`:

```
/home/jeff/projects/sandia/haero/chem_driver/chem_driver.cpp:430:17: error: no matching function for call to ‘Kokkos::View<double*, Kokkos::LayoutRight, Kokkos::OpenMP, void>::View(float*&, const ordinal_type&)’
  430 |   auto ropRev = RealType1DViewType(w, kmcd.nReac);
```

I've found hard-wired `double` typedefs in TChem in a few locations. For example, in `ext/TChem/src/core/TChem_Util.hpp`, line 70. I'm not sure why I haven't seen this before, but it's something we should raise as an issue to the TChem team.

@mschmidt271

--- a/chem_driver/chem_driver.hpp
+++ b/chem_driver/chem_driver.hpp
@@ -17,7 +17,6 @@ namespace chem_driver {
 
 // some aliases
 using ordinal_type = TChem::ordinal_type;
-using Real = haero::Real;
 using real_type_1d_view = TChem::real_type_1d_view;
 using real_type_2d_view = TChem::real_type_2d_view;
 using policy_type = typename TChem::UseThisTeamPolicy<TChem::exec_space>::type;
@@ -157,7 +156,7 @@ class ChemSolver {
 // implementation of the toy problem
 namespace from_tchem {
 
-using Real = haero::Real;
+using Real = double;
 
 struct SourceTermToyProblem {
   template <typename KineticModelConstDataType>

